sidecarDockerrun.sh

```diff
@@ -142,8 +146,14 @@ mount_manifest() {
     # gen3-fuse mounts the files in /data/<hostname> dir
     if [ ! -d $IDP_DATA_PATH/$MOUNT_NAME ]; then
         echo "Mounting manifest at $IDP_DATA_PATH/$MOUNT_NAME"
-        if (! gen3-fuse -config=/fuse-config.yaml -manifest=$PATH_TO_MANIFEST -mount-point=$IDP_DATA_PATH/$MOUNT_NAME -hostname=$BASE_URL -wtsURL=$WTS_URL -wtsIDP=$IDP -access-token=$ACCESS_TOKEN >/proc/1/fd/1 2>/proc/1/fd/2); then
-            echo "gen3-fuse failed to mount."
+        if [ -z $ACCESS_TOKEN ]; then
```

Comment:
Should this `$ACCESS_TOKEN` variable be changed to `TOKEN_JSON[$IDP]`?

Reply:
Not really, what we really are checking for whether or not it's a standard (local) workspace pod, or an external pod. Only external pods get access token mounted.